build.gradle

```diff
@@ -451,6 +451,8 @@ project(':iceberg-pig') {
     compile project(':iceberg-core')
     compile project(':iceberg-parquet')
 
+    compile "org.apache.commons:commons-lang3"
```

Comment:
Is there more context on the update to commons-lang3? Was this just an undeclared dependency? Why not use commons-lang?

Reply:
`lang` is being considered as deprecated, and the linter complaints that we should move to `lang3`. Therefore I've added it as a dependency. We only use it for serializing, de-serializing, so I'm fine to suppress the error as well. WDYT?

Reply:
Sounds fine to me, but we'll need to make sure this doesn't affect anything in the runtime Jar.

Reply:
Confirmed that this is provided by Spark and is excluded from the runtime Jar.